.travis.yml

```diff
@@ -25,15 +25,19 @@ branches:
   - /v(\d+)\.(\d+)\.(\d+)/
 env:
   - PYTHON="" MIMI_RUN_DEPENDENCY_TESTS="TRUE"
+before_script:
+  - julia -e 'using Pkg; Pkg.Registry.add(RegistrySpec(url ="https://github.com/mimiframework/MimiRegistry.git"))'
```

Comment:
Try to change this to:
```
- julia -e 'using Pkg; Pkg.update(); Pkg.Registry.add(RegistrySpec(url ="https://github.com/mimiframework/MimiRegistry.git"))'
```
The problem here is that if we add a new registry _before_ we've ever used `General`, the latter is not added to the registries at all.

Reply:
I'll try that!  Also just noting the tests won't pass until we remove appveyor, or add this line to the appveyor.yml file somehow!